lib/dialects/sqlite3/schema/ddl.js

```diff
@@ -344,6 +344,40 @@ class SQLite3_DDL {
     );
   }
 
+  setNullable(column, isNullable) {
+    return this.client.transaction(
```

Comment:
does sqlite even support transactions?

Reply:
It does support read transactions, which should be sufficient for an operation which would merely generate SQL statements

Reply:
The function is just a modification of the `alterColumn` function in the same file, which also uses transactions, so one would hope so!